content/services/courses/cpsc-221.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Basic Algorithms and Data Structures
+layout: course
+---
+
+> An introductionary course into algorithms and data structures, such as sorting algorithms, and linked list. The class covers the basics, 
+so not a lot of prior knowledge is required. Most importantly, this is the first class that introduces asymptotics notation (big O notation, like `O(n)`)
+and using them for algorithm analysis formally, so this could be handy for job interviews. You also learn about C++ and pointers.
+>
+> <cite><a href="https://github.com/Frozemint">Current CPSC 221 TA</a></cite>
```

Comment:
Instead of current, can you put a date? That way you don't need to tweak this in the future
```suggestion
> <cite><a href="https://github.com/Frozemint">2020 CPSC 221 TA</a></cite>
```

Reply:
A very good point indeed. I have added the suggestion.